widgetssdk/src/main/java/com/glia/widgets/chat/ChatView.kt

```diff
@@ -976,7 +987,7 @@ class ChatView(context: Context, attrs: AttributeSet?, defStyleAttr: Int, defSty
     private fun handleCaptureImageActionResult() {
         controller?.apply {
             photoCaptureFileUri?.also {
-                FileHelper.fixCapturedPhotoRotation(context, it)
+                context.fixCapturedPhotoRotation(it)
```

Comment:
In my opinion, it's more logical to keep this functionality in FileHelper rather than as extension function for Context.

Reply:
fixed - [cf6761a](https://github.com/salemove/android-sdk-widgets/pull/561/commits/cf6761ad6ab810bf0926f5c4b8f350825ca881f8)